server/src/organizations/organizations.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import { OrganizationsService } from './organizations.service';
+import { CreateOrganizationDto } from './dto/create-organization.dto';
+import { UpdateOrganizationDto } from './dto/update-organization.dto';
+
+
+@Controller('organizations')
+export class OrganizationsController {
+  constructor(private readonly organizationsService: OrganizationsService) {}
+
+  @Post()
+  create(@Body() createOrganizationDto: CreateOrganizationDto) {
```

Comment:
TS return types!
checkout the controllers of other resources that have been added and i think you'll be able to figure it out
i.e. [this](https://github.com/Nonprofit-Exchange-Hub/web-app/blob/19733511b87e8dcbe16b05e48530b8651f50fbb2/server/src/assets/assets.controller.ts#L44)
format is `function name(arg: argType): functionReturnType {...`

Reply:
done!